lib/mayaUsd/fileio/jobs/meshDataReadJob.cpp

```diff
@@ -63,8 +63,6 @@ GfMatrix4d getTransform(const UsdPrim& prim, const std::map<SdfPath, GfMatrix4d>
 
 void getComponentTags(MFnMeshData& dataCreator, const UsdGeomMesh& mesh)
 {
-#if MAYA_API_VERSION >= 20220000
```

Comment:
Since Maya 2022 is the lowest Maya version we now support I removed any ifdef not needed anymore.